component/prometheus/exporter/windows/windows_test.go

```diff
@@ -10,39 +11,40 @@ import (
 var (
 	exampleRiverConfig = `
 		enabled_collectors = ["textfile","cpu"]
+		timeout = "1m"
 		
 		exchange {
 			enabled_list = ["example"]
 		}
 		
 		iis {
-			site_whitelist = ".+"
-			site_blacklist = ""
-			app_whitelist = ".+"
-			app_blacklist = ""
+			site_include = ".+"
```

Comment:
This is the preferred nomenclature but we should allow the old values and mark them as deprecated.

Reply:
The upstream windows_exporter leaving deprecation notes. 

https://github.com/prometheus-community/windows_exporter/blob/79781c6d75fc5cef4d28685b17a4b581e038aa10/collector/net.go#L90

I hope thats enough